pages/treeherder.py

```diff
@@ -93,6 +93,9 @@ class ResultSet(PageRegion):
         _datestamp_locator = (By.CSS_SELECTOR, '.result-set-title-left > span a')
         _jobs_locator = (By.CLASS_NAME, 'job-btn')
         _pin_all_jobs_locator = (By.CLASS_NAME, 'pin-all-jobs-btn')
+        _dropdown_toggle_locator = (By.CSS_SELECTOR, 'button.btn.btn-sm.btn-resultset.dropdown-toggle')
+        _top_of_range_link_locator = (By.LINK_TEXT, 'Set as top of range')
```

Comment:
Locators should by styled using our style guide: https://wiki.mozilla.org/QA/Execution/Web_Testing/Docs/Automation/StyleGuide#Locators

This locator would use either CLASS_NAME or CSS_SELECTOR.